share/man/man9/intro.9

```diff
@@ -33,7 +33,7 @@ the first time will be able to understand.
 .Pp
 To further set expectations, we acknowledge that kernel documentation, like the
 source code itself, is forever a work-in-progress.
-There will be large sections of the codebase whose documentation is subtly or
+There will be large sections of the codebase where documentation is subtly or
 severely outdated, or missing altogether.
```

Comment:
I'd further reduce this, to: 

> For some of the codebase, documentation will be subtly or severely outdated, or missing.

(The preceding sentence sets the stage. This sentence can be less dramatic.)

Reply:
I don't hate it, but it is a matter of taste, I suppose.

Out of anywhere in the in the whole section, the introductory paragraphs should be written with a distinct voice. I think the slight dramatic-ism is consistent with how I want it to be read, and is justified here. Of course I wrote it and will not be able to see it from the eyes of a first-time reader.